pkg/schedule/schedulers/hot_region_test.go

```diff
@@ -533,11 +535,14 @@ func checkHotWriteRegionScheduleByteRateOnly(re *require.Assertions, enablePlace
 		ops, _ := hb.Schedule(tc, false)
 		op := ops[0]
 		clearPendingInfluence(hb.(*hotScheduler))
-		re.Equal(4, op.Len())
 		if op.RegionID() == 2 {
+			re.Equal("move-hot-write-peer", op.Desc())
+			re.Equal(4, op.Len())
 			// peer in store 1 of the region 2 can transfer to store 5 or store 6 because of the label
 			operatorutil.CheckTransferPeerWithLeaderTransferFrom(re, op, operator.OpHotRegion, 1)
 		} else {
+			re.Equal("move-hot-write-leader", op.Desc())
+			re.Equal(5, op.Len())
```

Comment:
why is the length different?

Reply:
https://github.com/tikv/pd/issues/7872 Move leader without joint will appear as an extra transfer leader, I'll try to adjust the test firstly to make it look more reasonable